src/main/components/Browse/Browse.js

```diff
@@ -52,12 +157,14 @@ class Browse extends Component {
 
     const { analysis, selectAnalysis } = this.props;
     const dashboard = this.props.data.dashboards[0];
-    const analyses = this.state.analyses;
+    const { analyses, chosenFilters } = this.state;
 
     return (
       <div>
         <IconButton
-          onClick={() => this.setState({ isOpen: true })}
+          onClick={() =>
+            this.setState({ isOpen: true }, () => this.clearFilters())
```

Comment:
I worry about this. This is going to trigger at least 2 renders:
- 1 because you setState here
- 1 because there is a setState within clearFilters